source/getting-started/control-with-amdc/encoder-fb/index.md

```diff
@@ -89,22 +89,31 @@ double task_get_theta_m(void)
 
-To determine the appropriate offset value, eliminate any source of load torque on the shaft and apply a large current vector at 0 degrees ($I_u = I_0$, $I_v = I_w = -\frac{1}{2} I_0$). This should cause the rotor to align with the phase U winding axis. The offset value can now be obtained as `enc_theta_m_offset = encoder_get_position()`. Alternately, the user may also inject a current along the d-axis using the AMDC [Signal Injection](https://docs.amdc.dev/getting-started/user-guide/injection/index.html) module. While injecting d-axis current, the user must ensure that the rotor position is set to zero in the control code. 
+#### Step 1: Determine approximate offset
 
-After obtaining the offset, the user needs to set the variable `enc_theta_m_offset` to the appropriate value in the `task_get_theta_m()` function. 
+The approximate encoder offset can be found using the following simple procedure without feedback control:
 
-To ensure that the obtained encoder offset is correct, the user may perform additional validation. For a permanent magnet synchronous motor, this can be done as follows:
+1. Set the `enc_theta_m_offset` to 0 in the control code `task_get_theta_m()`.
+2. Eliminate any source of load torque on the shaft.
+3. Align the rotor with the phase U winding axis by applying a large current vector at 0 degrees ($I_u = I_0$, $I_v = I_w = -\frac{1}{2} I_0$). This could be accomplished by:
+    1) Using a DC power supply, or
+    2) Injecting a current command on the d-axis using the AMDC [Signal Injection](/getting-started/user-guide/injection/index.rst) module with `theta_m_enc` fixed to 0.
+4. Record the current encoder position and use this as the offset value: `enc_theta_m_offset = encoder_get_position();`.
+5. Update the variable `enc_theta_m_offset` to the appropriate value in `task_get_theta_m()`.
 
-1. Spin the motor up-to a steady speed under no load conditions
-1. Measure the d-axis voltage commanded by the current regulator
-1. Repeat the experiment for a few different rotor speeds
-1. Plot the d-axis voltage against the rotor speed
-1. The d-axis voltage should be close to zero for all speeds, if the offset is tuned correctly
-1. In-case there is an error in the offset value, a significant speed dependent voltage will appear on the d-axis voltage. In this case, the user may have to re-measure the encoder offset.
+#### Step 2: Determine precise offset
 
+Friction and cogging torque in the motor decrease the accuracy of the estimate in Step 1. The precise offset can be found by fine-tuning the `enc_theta_m_offset` from Step 1 while using closed-loop control to rotate the shaft at significant speed. This can be done as follows:
```

Comment:
Let's provide the d-axis voltage expression and have a sentence explaining how the technique works with respect to this (i.e., no loading so $i_q = 0$, we're commanding $i_d=0$, and therefore any voltage observed in the d axis must be due to encoder offset error).